### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by voronine**
- [DEMO LINK](https://voronine.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://voronine.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,61 @@
...
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="index.html" class="header__homelink">
```
Comment (Moroz-Dmytro) [inline src/index.html:16]:
```suggestion
        <a href="#l" class="header__homelink">
```

Comment (Moroz-Dmytro) [reply src/index.html:16]: image

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,61 @@
...
+          <img
+           src="./images/logo.png"
+           class="logo"
+           alt="logo"
+           >
+        </a>
+      </div>
+      <nav class="nav">
```
Comment (Moroz-Dmytro) [inline src/index.html:24]:
```suggestion
      </div>
      
      <nav class="nav">
```

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,61 @@
...
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item" >
+            <a class="nav__link"
+            data-qa ="hover"
+            href="#laptops&computers"
+            >
+            Laptops & Computers
+            </a>
```
Comment (Moroz-Dmytro) [inline src/index.html:44]:
```suggestion
            <a
              class="nav__link"
              data-qa ="hover"
              href="#laptops&computers"
            >
              Laptops & Computers
            </a>
```

Comment (Moroz-Dmytro) [reply src/index.html:44]: image

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+body,
+header,
+ul,
+li,
+nav,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --color-hover: #00acdc;
```
Comment (Moroz-Dmytro) [inline src/style.css:10]: image

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+  position: relative;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
```
Comment (Moroz-Dmytro) [inline src/style.css:42]: you can set this as default font styles for your page (under html selector)

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Good work 🔥 
Only small comments to make it perfect

File: src/index.html [hunk 6]
```diff
@@ -6,9 +6,62 @@
...
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#1" class="header__homelink">
+          <img
+           src="./images/logo.png"
+           class="logo"
+           alt="logo"
+            >
```
Comment (BudnikOleksii) [inline src/index.html:21]:
```suggestion
          <img
             src="./images/logo.png"
             class="logo"
             alt="logo"
          >
```
Checklist:
13. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

File: src/index.html (hunk 3 above)
Comment (BudnikOleksii) [reply src/index.html:45]: Fix in accordance with the suggestion.

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Pay attention to comments and the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,71 @@
-body {
+html {
+  --color-hover: #00acdc;
```
Comment (dffUqp) [inline src/style.css:2]: good practice to declare variables inside the `:root` selector
```css
:root {
  --hover-color: #00acdc;
}
```

File: src/index.html [hunk 8]
```diff
@@ -6,9 +6,63 @@
...
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#1" class="header__homelink">
```
Comment (dffUqp) [inline src/index.html:16]: image
```suggestion
        <a href="#home" class="header__homelink">
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,71 @@
...
+
+.is-active::after {
+  display: block;
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--color-hover);
+  border-radius: 8px;
```
Comment (dffUqp) [inline src/style.css:70]: border-radius should be half of width/height. If you set bigger value it is not a good practice.

**Review by dffUqp — CHANGES_REQUESTED** [review]
LGTM:fire: but needs a few minor fixes:smiley:

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Nice job👍🏻
But a few comments are still not fixed.
Pay attention to review comments.

File: src/index.html (hunk 8 above)
Comment (BudnikOleksii) [reply src/index.html:16]: Pay attention to comments.

Comment (voronine) [reply src/index.html:16]: Yesterday there was this comment, initially, I had "index.html" set up, and I thought it was the home page. I changed it to "#home," hoping it's correct.

**Review by BudnikOleksii — APPROVED** [review]
Great🚀

**Pull request closed without merging**